### DISCUSSION TITLE
Toast onPause, onResume doubts

### DISCUSSION THREAD

**Discussion by emigdio821** [radix-ui/primitives#1950] [Help] [answered]
Hi there, I recently started using RadixUI, Im trying to add a countdown in each toast, I saw `onPause` and `onResume` in the documentarion but I was not able to see examples, is there any simple example on how to use these functions? thanks in advance.

**Comment by joaom00** [comment, marked as answer]
Hi @emigdio821,

`onPause()` is called when:

- The pointer is moved over the toast's viewport
- The toast's viewport is focused
- The window is blurred

`onResume()` is called when:

- The pointer is moved away from the toast's viewport
- The toast's viewport is blurred
- The window is focused

In this [preview](https://o0k4fd-5173.preview.csb.app/) you can see the logs in the console

[CodeSandbox](https://codesandbox.io/p/sandbox/stupefied-water-o0k4fd)

**Reply by emigdio821** [reply]
Thank you @joaom00 !
